catalog/files/impl/src/main/java/org/projectnessie/catalog/files/adls/AdlsObjectIO.java

```diff
@@ -144,7 +144,9 @@ public void configureIcebergTable(
     }
 
     icebergConfigDefaults(config);
-    icebergConfigOverrides(storageLocations, config);
+    AdlsNamedFileSystemOptions fileSystemOptions = icebergConfigOverrides(storageLocations, config);
+
+    fileSystemOptions.tableConfigOverrides().forEach(config);
```

Comment:
Could be done in `icebergConfigOverrides`

Reply:
good point

Reply:
fixed